pkg/verifier/cosign/cosign.go

```diff
@@ -594,3 +623,27 @@ func processAKVSignature(sigEncoded string, staticSig oci.Signature, publicKey c
 	}
 	return hashType, staticSig, nil
 }
+
+// verificationMessage returns a string list of all verifications performed
+// based on https://github.com/sigstore/cosign/blob/5ae2e31c30ee87e035cc57ebbbe2ecf3b6549ff5/cmd/cosign/cli/verify/verify.go#L318
+func verificationMessage(bundleVerified bool, co *cosign.CheckOpts) []string {
+	var messages []string
+	if co.ClaimVerifier != nil {
+		if co.Annotations != nil {
+			messages = append(messages, annotationMessage)
+		}
+		messages = append(messages, claimsMessage)
+	}
+	if bundleVerified {
+		messages = append(messages, offlineBundleMessage)
+	} else if co.RekorClient != nil {
+		messages = append(messages, rekorClaimsMessage)
+		messages = append(messages, rekorSigMessage)
+	}
+	if co.SigVerifier != nil {
+		messages = append(messages, sigVerifierMessage)
+	} else {
+		messages = append(messages, certVerifierMessage)
```

Comment:
question: in cosign lib, it prints the info when `fulcioVerified` is true, is it always true in Ratify usage?

Reply:
Yeah great question. The reason is that in the cosign lib the `fulcioVerified` = true if `co.SigVerifier == nil` https://github.com/sigstore/cosign/blob/5ae2e31c30ee87e035cc57ebbbe2ecf3b6549ff5/cmd/cosign/cli/verify/verify.go#L285